private-templates-service/README.md

```diff
@@ -68,29 +78,33 @@ app.use(function (req, res, next) {
     "GET, POST, OPTIONS, DELETE"
   );
   next();
-});`
-
-**4. In the `adaptivecards-templates/private-templates-service/server/app.ts` file, replace ` "#{DB_CONNECTION_TOKEN}#"` with the real DB connection string. Make sure to NOT commit this file.**
-
-**5. Under the `adaptivecards-templates/private-templates-service/server`, run `npm run link-client`. Please note you may have to run  `sudo npm run link-client` if you are having errors with permissions.**
+});
+```
 
-**6. Inside of `adaptivecards-templates/private-templates-service/server` run `npm install`, run `tsc`, and run `npm run dev` to launch the client and server in your default browser.**
+5. `cd adaptivecards-templates/private-templates-service/server` and run `npm run init-app`. This installs and links dependencies.
 
-**You're good to go!**
+6. Run `npm run dev`. This command concurrently runs the client and server locally. Navigate to `localhost:3000` to see the site.

```

Comment:
Same thing as comment below: 

Can we say `In the same directory as Step 6, run npm run dev... ?